### DISCUSSION TITLE
Apply existing model to other series

### DISCUSSION THREAD

**Discussion by M-Adrian-N** [ourownstory/neural_prophet#1197] [Q&A - get help using NeuralProphet] [answered]
Hi,
How can I use existing fitted models (I've created) to create forecasts for other sets of data? 
Example:
Say I have 10k items to create models for, time constraints don't allow me to create a model for every item, so then I will instead create a model for its Product Family (that are only 100), in which 99% of its items follow the same trend/seasonality. 

Now how can I run the item level time series through the product group family fitted models to extract item level forecasts? Which should basically mimic the product group family seasonality/trend by of course applied to that item's specific values.

Something equivalent to what I'm asking would be AutoARIMA.forward

**Comment by Kevin-Chen0** [comment, marked as answer]
^ @alfonsogarciadecorral
